README.md

```diff
@@ -164,7 +117,6 @@ deployment.apps/csi-addons-controller-manager created
 
 The [Contribution Guidelines](CONTRIBUTING.md) contain details on the process
 to contribute to this project.
-
```

Comment:
Is this change required?

Reply:
Not required, just removed the extra space.